### DISCUSSION TITLE
ElegantMustard Overlay AIO Package

### DISCUSSION THREAD

**Discussion by lscambo13** [lscambo13/ElegantMustard#3] [General]
### **This is an all-in-one package including the major versions of ElegantMustard Statistics Overlay for the sake of simplicity.**
**For detailed instructions, make sure to read the [README.md](https://github.com/lscambo13/ElegantMustard/blob/main/README.md) file.**

### **Version 1 (Inspired from Returnal)**
**This release is inspired from the benchmark tool in Returnal .**
```
This version needs MSI Afterburner to be setup.
There are three variants: Full, Compact and Minimal.
```

### **Version 4 (Inspired from Steam Deck)**
**This release is inspired from Steam Deck statistics top bar.**
```
This version does **not** need MSI Afterburner to be setup.
However, it has to be set up in Rivatuner aka RTSS.
There is only one variant.
```

### **Version 5.1 (Experimental)**
```
-Same as version 4 with some changes
-RTSS/Rivatuner only
-Removed GPU fan speeds
-Adjusted GPU and CPU temperature ranges for dynamic color (ranges 0-76-79-100)
-Added CPU core speeds (only 12 threads for now)
-Fixed CPU per-core usage colors based on temperature (ranges 0-85-95-100)
```

### **For detailed instructions, make sure to read the [README.md](https://github.com/lscambo13/ElegantMustard/blob/main/README.md) file.**
**For previous changelogs and versions, read the [Releases](https://github.com/lscambo13/ElegantMustard/releases)  section.**

<hr /><em>This discussion was created from the release <a href='https://github.com/lscambo13/ElegantMustard/releases/tag/v5.1'>ElegantMustard Overlay AIO Package</a>.</em>

**Comment by rakrimes** [comment]
Where do I put the file I downloaded, I picked the Full AIO

**Reply by lscambo13** [reply]
Open Rivatuner > Overlay Editor
Click File > Import
Select your desired overlay

There is also additional setup needed; you should read the README.MD file too.

**Comment by rakrimes** [comment]
Yep read the readme just couldn’t figure out the install

**Reply by lscambo13** [reply]
No problem.

Try it, and feel free to ask questions if you face any trouble.

**Comment by azaelbr** [comment]
Hello, how are you, my friend? When I load your overlay, if I have AIDA64 open, it messes everything up by endlessly pulling all the data from AIDA64.
How can I fix this?

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/3e5b5587-3617-49be-9cc4-6d904bf28d84" />

**Reply by lscambo13** [reply]
> Hello, how are you, my friend?

I'm well. Thank you for asking.

> if I have AIDA64 open, it messes everything up

I installed AIDA64 to test it, but I couldn't reproduce this issue, but I think it can be deleted.

Have you tried selecting this group and deleting it using the physical keyboard key?

<img width="178" height="124" alt="image" src="https://github.com/user-attachments/assets/14647add-b93b-4a2c-b7f2-6a7364596315" />

**Reply by azaelbr** [reply]
Every time I delete the group that contains these texts, it moves to the next layer until there's nothing left in the project

**Comment by WistopXD** [comment]
In my game this looks huge. How can I resize it?
<img width="1394" height="1039" alt="Screenshot_33" src="https://github.com/user-attachments/assets/8041c7d4-589b-4978-b798-83d4592178ca" />
<img width="1919" height="1043" alt="Screenshot_34" src="https://github.com/user-attachments/assets/fd810203-fd60-45d3-83c3-08a9114b69fc" />
